demos/python_demos/human_pose_estimation_3d_demo/pose_extractor/wrapper.cpp

```diff
@@ -79,14 +79,13 @@ PyModuleDef pose_extractor_module = {
 };
 
 PyMODINIT_FUNC PyInit_pose_extractor(void) {
-    PyObject* module = PyModule_Create(&pose_extractor_module);
-    if (module == nullptr) {
-        return nullptr;
-    }
     import_array();
     if (PyErr_Occurred()) {
         return nullptr;
     }
-
+    PyObject* module = PyModule_Create(&pose_extractor_module);
+    if (module == nullptr) {
+        return nullptr;
+    }
```

Comment:
Looks unnecessary. You can just return `module` in all cases. In fact, you don't even need the `module` variable.